__tests__/app.test.js

```diff
@@ -28,4 +28,16 @@ describe("CORE:",() => {
 
         })
     })
+
+    it("200: returns all API endpoints", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then((response) => {
+            const {
+                body: { endpoints },
+            } = response;
+            expect(typeof endpoints).toBe("object")
```

Comment:
This could need more testing.
You could require in the endpoint object and assert that it looks the same as the response from the endpoint.